### PR TITLE
Fix `Option<Bytes>` encoding

### PR DESCRIPTION
**Pull request by adamdossa**
Correctly encoded `Option<Bytes>` by pre-pending 1 for valid options.

### CODE REVIEW

File: packages/react-params/src/Param/BaseBytes.tsx [hunk 1]
```diff
@@ -101,6 +102,10 @@ function BaseBytes ({ asHex, children, className = '', defaultValue: { value },
         value = compactAddLength(value);
       }
 
+      if (isInOption && isValid) {
```
Comment (jacogr) [inline packages/react-params/src/Param/BaseBytes.tsx:105]: `isInOption` just needs to be added on line 118 (dependency)

File: packages/react-params/src/Param/BaseBytes.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ import React, { useCallback, useState } from 'react';
 import styled from 'styled-components';
 
 import { CopyButton, IdentityIcon, Input } from '@polkadot/react-components';
-import { compactAddLength, hexToU8a, isAscii, isHex, stringToU8a, u8aToHex, u8aToString, u8aToU8a } from '@polkadot/util';
+import { compactAddLength, hexToU8a, isAscii, isHex, stringToU8a, u8aToHex, u8aToString, u8aToU8a, u8aConcat } from '@polkadot/util';
```
Comment (jacogr) [inline packages/react-params/src/Param/BaseBytes.tsx:11]:
```suggestion
import { compactAddLength, hexToU8a, isAscii, isHex, stringToU8a, u8aConcat, u8aToHex, u8aToString, u8aToU8a } from '@polkadot/util';
```

Ordering, lint is complaining.

**Review by jacogr — COMMENTED** [review]
Thank you. Some small comments from linting, otherwise looks sane.

**Comment by adamdossa** [comment]
@jacogr looks good to go when you're ready.

**Review by jacogr — APPROVED** [review]
Thank you. Looks sane.

**Pull request merged**